doc/develop/flash_debug/probes.rst

```diff
@@ -127,7 +127,10 @@ This debug probe is compatible with the following debug host tools:
 Once the MCU-Link host tools are installed, the following steps are
 required to program the CMSIS-DAP firmware:
 
-1. Put the MCU-Link microcontroller into DFU boot mode by attaching the DFU
+1. Make sure the MCU-Link utility is installed on your host machine. This can
+   be installed by installing :ref:`linkserver-debug-host-tools`.
```

Comment:
"installed by installing" is odd :) maybe:

```suggestion
1. Make sure the MCU-Link utility is present on your host machine. This can
   be done by installing :ref:`linkserver-debug-host-tools`.
```

and do something for the 4 other occurences?